measurementWriter.go

```diff
@@ -1,14 +1,25 @@
 package main
 
-import "fmt"
+import (
+	"context"
+	"github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
+	"strconv"
+	"time"
+)
 
-type MeasurementPrintWriter struct {
+type MeasurementWriter struct {
+	writeApi    api.WriteAPIBlocking
 	measurement chan Measurement
 }
 
-func (mw *MeasurementPrintWriter) ReadMeasurement() error {
+func (mw *MeasurementWriter) StoreMeasurement() {
 	for {
 		m := <-mw.measurement
-		fmt.Println("Id:", m.Id, "Value: ", m.Value)
+		p := influxdb2.NewPointWithMeasurement("deviceValues").
+			AddTag("id", strconv.Itoa(m.Id)).
+			AddField("value", m.Value).
+			SetTime(time.Now())
+		mw.writeApi.WritePoint(context.Background(), p)
```

Comment:
This method return an error. You should at least log the error.

In general, this process is accepting new dataPoints over AMQP - the whole process should be durable. In real scenario we would setup manual acknowledgements on AMQP, check influx write error here, reject the message if this is temporary issue and/or panic the whole process if for example influxdb is down (because the process is useless if it cannot write to influx, maybe restart will help). In case of this simple scenario let's just log the error but keep in mind the bigger picture because this is the way we would handle the issue on production.